scene/3d/light.cpp

```diff
@@ -156,9 +156,8 @@ void Light::_update_visibility() {
 	if (!is_inside_tree())
 		return;
 
-	bool editor_ok = true;
-
 #ifdef TOOLS_ENABLED
+	bool editor_ok = true;
```

Comment:
I don't see the point of this change.

Reply:
Actually I think that whole method could be removed, it doesn't do anything.

Reply:
The whole `editor_only` API seems to be useless actually, would need input from @reduz to know if it should be fixed, or removed.

Reply:
Maybe leave out those changes for now, I'll see to either remove it all or implement what's missing.

Reply:
Will undo the changes then.